src/components/CookieWarning.js

```diff
@@ -36,21 +32,28 @@ export const CookieWarning = () => {
     }, cookies)
     signOut()
   }
-  return !(acceptedCookies) && div({
+
+  return !cookiesAccepted && div({
     style: {
       position: 'fixed', height: 100, bottom: 0, width: '100%',
       display: 'flex', alignItems: 'center', justifyContent: 'space-between',
-      backgroundColor: 'white', borderTop: `6px solid ${colors.primary()}`
+      backgroundColor: colors.dark(0.15), borderTop: `6px solid ${colors.primary()}`,
+      // Minimum supported page width
+      minWidth: 1200
```

Comment:
is this the minimum supported width for Terra? I did notice that if a user has a width of 1000px then they cannot see or scroll to the accept button. (I could not scroll the banner to the side)
![cant see the buttons scrolled all the way to the right 1000px window](https://user-images.githubusercontent.com/5375000/106306617-50e1f200-622c-11eb-9cb6-59e1056df853.png)

We don't support mobile devices right? I won't go there :)

Reply:
We do not support mobile devices. The inability to scroll it at a smaller width is a problem, I will fix that. 1200px is our min-width, as set on the body file.